lib/streamlit/runtime/forward_msg_queue.py

```diff
@@ -33,6 +33,15 @@ class ForwardMsgQueue:
     a single thread.
     """
 
+    _before_enqueue_msg = None
```

Comment:
nit: add the same type hint you have defined in the `on_before_enqueue_msg` function: _before_enqueue_msg: Callable[[ForwardMsg], None] | None = None